naga/src/front/wgsl/error.rs

```diff
@@ -775,10 +775,10 @@ impl<'a> Error<'a> {
                 ]
             },
             Error::ExceededLimitForNestedBraces { span, limit } => ParseError {
-                message: "brace nesting limit reached".into(),
-                labels: vec![(span, "limit reached at this brace".into())],
+                message: "Statements nested too deeply".into(),
```

Comment:
issue: The capitalized casing, for better or worse, is _not_ consistent with other diagnostics.

```suggestion
                message: "statements nested too deeply".into(),
```

Reply:
This feedback also applies to the `notes` field.